identity-wallet/resources/jsonschemas/README.md

```diff
@@ -0,0 +1,11 @@
+This folder contains the Json schemas of 3 digital credential data format:
```

Comment:
According to the [official website](https://json-schema.org), the name of this specification is "**JSON Schema**" instead of "Json schema" (or similar). The official term is also used by other credible sources such as [Postman](https://blog.postman.com/what-is-json-schema/), [MongoDB](https://www.mongodb.com/resources/languages/json-schema-examples) and the [project on GitHub itself](https://github.com/json-schema-org/json-schema-spec), so I think we should follow this as well.